Dockerrun.aws.json

```diff
@@ -0,0 +1,12 @@
+{
+  "AWSEBDockerrunVersion": "1",
+  "Image": {
+    "Name": "276375911640.dkr.ecr.us-east-1.amazonaws.com/pwa-demo:latest",
```

Comment:
this tag will need to update based on semantic version tagging

Reply:
this is modified in each build script to be either the pr id tag or the commit SHA